controller.js

```diff
@@ -10,10 +10,21 @@ const getCategories = (request, response, next) => {
 }
 
 const getReviews = (request, response, next) => {
-    fetchReviews()       
+    const {order_by} = request.query
+    let {category} = request.query
+    const {sort_by} = request.query
+
```

Comment:
Nice destructuring, any reason category uses `let` rather than `const`?

Reply:
allows me to reassign category to the result from my checkCategories function, got an error when that was const